test/kldivergences.jl

```diff
@@ -0,0 +1,69 @@
+@testset "KL divergences" begin
+    function test_kl(p, q)
+        @test kldivergence(p, q) > 0
+        @test kldivergence(p, q) ≈ Distributions.mcexpectation(p, x -> let p = pdf(P,x); (p > 0) * log(p / pdf(Q, x)) end; n_samples=100_000)
```

Comment:
This should be updated. It might also be better to just call the fallback method with `invoke` instead of reimplementing it here.